src/rules/incognia-authentication.js

```diff
@@ -15,9 +15,9 @@
 async function incogniaAuthenticationRule(user, context, callback) {
   const _ = require('lodash@4.17.19');
 
-  const { IncogniaAPI } = require('@incognia/api@1.0.0');
+  const { IncogniaAPI } = require('@incognia/api@1.1.0');
```

Comment:
You will need to require v1.1.1
```suggestion
  const { IncogniaAPI } = require('@incognia/api@1.1.1');
```